frontend/src/modules/home/quiz/Quiz.js

```diff
@@ -36,12 +59,13 @@ export default function Quiz() {
 
     return (
         <div className={classes.root}>
+            <ComponentA answered={answered} goNext = {goNext} question ={question}/>
```

Comment:
Fix the formatting here:
```suggestion
            <ComponentA answered={answered} goNext={goNext} question={question}/>
```